tests/test_document.py

```diff
@@ -1,26 +1,21 @@
 from __future__ import absolute_import, unicode_literals
-
+import pytest
 from six import string_types
-
 from c8.exceptions import (
-    DocumentCountError,
     DocumentDeleteError,
     DocumentGetError,
-    DocumentInError,
     DocumentInsertError,
     DocumentReplaceError,
     DocumentRevisionError,
     DocumentUpdateError,
```

Comment:
DocumentUpdateError and DocumentReplaceError might not be required here as we are doing string comparision now for the error. So maybe you can just check if it still needs to be imported from c8.exceptions or not. Also check for other imports which are not required or are not accessed

Reply:
According to Pycharm, all the imports are being used. Regarding the exception type, we have kind of a mix, some functions are making string comparisons, but others are catching the exception type